doc/contributing/testing_ruby.md

```diff
@@ -79,6 +79,12 @@ We can run any of the make scripts [in parallel](building_ruby.md#label-Running+
     make test-all TESTS=-v
     ```
 
+    We can display the help of the `TESTS` option:
+
+    ```
+    $ make test-all TESTS=--help
+    ```
```

Comment:
Also `ruby test/runner.rb --help` can, if your ruby is 2.7 or later.

Reply:
Thanks for the info. Do you have an idea about how to add the `ruby test/runner.rb --help` to this document?

Like this? I suppose that adding the runtime `ruby` version requires 2.7 or later is a bit too detailed. 

```
diff --git a/doc/contributing/testing_ruby.md b/doc/contributing/testing_ruby.md
index 167ab55c55..1289cd4341 100644
--- a/doc/contributing/testing_ruby.md
+++ b/doc/contributing/testing_ruby.md
@@ -85,6 +85,12 @@ We can run any of the make scripts [in parallel](building_ruby.md#label-Running+
     $ make test-all TESTS=--help
     ```
 
+    We can also display the help with
+
+    ```
+    $ ruby test/runner.rb --help
+    ```
+
     If we would like to run both the `test/` and `bootstraptest/` test suites, we can run
 
     ```
```